tests/qos/Pfc_Storm_with_Shared_Headroom_test_plan.md

```diff
@@ -19,11 +19,11 @@ The test checks if any PFC Frames are sent to the peer link from the DUT port.
    - Make sure to send atleast num_pkts_pfs_frame + private_headroom pkts pkts
    - num_pkts_pfs_frame: num of pkts required to be sent in order to trigger a PFC frame from the DUT. More on this here: https://github.com/Azure/sonic-mgmt/blob/master/tests/qos/files/mellanox/qos_param_generator.py
    - private_headroom_pkts is specific to mellanox which is in the order of a few pkts.
-   - Check the PFC Rx Counters to see verify if the occupancy has indeed crossed into 
+   - Check the PFC Tx Counters to see verify if the occupancy has indeed crossed into the shared headroom region
```

Comment:
This should be the "show priority-group watermark headroom" counters that will indicate if there is some occupancy in the headroom. pfc counters will only give the number of pfc frames sent/received.

Reply:
pfc tx counters will show the cumulative value of pfc frames that were transmitted by the device since the last clear. pfc frames are transmitted when the shared space is exceeded. there should be some packets that come in after this condition is met so as to have occupancy in the shared headroom. since in the tests, we are just calculating the number of packets beforehand to ensure that they get trickled into shared headroom, this info is not important. but if we were to do  manual tests , we should start off with clearing watermark counters and then continuing to send packets till we see some occupancy in "show priority-group watermark headroom" counters